components/WalletConnectButton.tsx

```diff
@@ -0,0 +1,13 @@
+import { WalletMultiButton } from '@solana/wallet-adapter-react-ui';
+
+const WalletConnectButton = () => {
+  return (
+    <div>
+      <div className="w-full mx-1 text-primary hover:text-secondary">
```

Comment:
Styles object. looool. Sorry.

Also, you have an outer div before this. Any reason for that?

Reply:
ahhh forgot again sorry i'll make sure to double check next time!